documentation/modules/auxiliary/gather/advantech_webaccess_creds.md

```diff
@@ -0,0 +1,32 @@
+## Description
+
+This module allows you to log into Advantech WebAccess, and gather credentials from the user list.
```

Comment:
Sure, I'll update it. Thanks!

I also tried to look for CVEs for these problems, and the closest I found was this: https://www.cvedetails.com/cve/CVE-2016-0853/

However, as far as I can tell, that particular CVE is for older versions for WebAccess. Not 8.1. So my guess these problems could earn their own CVEs